federatedscope/core/configs/cfg_training.py

```diff
@@ -22,6 +22,7 @@ def extend_training_cfg(cfg):
     cfg.optimizer.type = 'SGD'
     cfg.optimizer.lr = 0.1
     cfg.optimizer.weight_decay = .0
+    cfg.optimizer.mmt = .0
```

Comment:
I'm not sure the abbreviation for momentum is general enough.

Reply:
changed to "momentum" accordingly